AuxTel/AuxTel-Standard-Operations/AuxTel-Daytime-Operations/latiss-daily-calibrations-full.rst

```diff
@@ -0,0 +1,277 @@
+- If ``generate_calibrations`` is ``True``, a (daily) combined calibration image per detector will be certified in a `butler`_ ``CALIBRATION`` `collection`_.
+
+
+.. _LATISS-Daily-Calibrations-Full-Procedure-Steps:
+Procedure Steps
+===============
+
+1. Enable ATCS and LATISS using the standard scripts :file:`enable_atcs.py` and :file:`enable_latiss.py` with no configuration. 
+2. Enable ``Scheduler:2`` with a valid scheduler configuration. Use the standard script :file:`auxtel/scheduler/enable.py` with the configuration required for the run and available in the corresponding night log. 
+3. Run the script :file:`add_block.py` to the ATQueue  with the following configuration:
+
+.. code-block:: text
+  :caption: add_block.py
+
+  id: latiss_daily_calibrations
+
+
+The BLOCK with `latiss_daily_calibrations` configuration will queue the following scripts:
+
+The BLOCK will start with the `set_summary_state.py` script to enable ATWhiteLight CSC with the following configuration:
+
+.. code-block:: text
+  :caption: set_summary_state.py
+ 
+  data:
+     -
+       - ATWhiteLight 
+       - ENABLED 
+
+The `set_summary_state.py` will enable ``OCPS:1`` CSC.
+
+.. code-block:: text
+  :caption: set_summary_state.py
+ 
+  data:
+     -
+       - OCPS:1
+       - ENABLED
+
+The SAL script `power_on_atcalsys.py` with no configuration inserted into the BLOCK `latiss_daily_calibrations` structure will start the chiller, turn on the white light and open the shutter.  This script takes 15 minutes to complete, the time it takes to warm up the white light. On Chronograf using the query :file:`lsst.sal.ATWhiteLight.logevent_logMessage.message`, the lamp reports its retry loop status explicity.
+
+.. code-block:: text
+  :caption: power_on_atcalsys.py
+
+
+The `prepare_for_flat.py` script (empty configuration) will position the telescope and dome in FLAT position. The telescope will point towards the dome flat screen (mount Az = 188.7 deg, mount El = 39.0 deg, dome Az= 2.59 deg). Confirm in the cameras aux-cam01 or aux-cam02 that the white light is on, and telescope is pointing to the dome flat screen.
+
+.. code-block:: text
+   :caption: prepare_for_flat.py
+
+The `make_latiss_calibrations` script for each filter (SDSSr_65mm, SDSSg_65mm, SDSSz_65mm, SDSSy_65mm and empty_1) will take seven sets of BIAS, DARKS and FLATS calibration images. Confirm images are showing up in `RubinTV`_. The exposure times have been adjusted for raw BIAS and DARK frames of about 10 ADUs. In the case of FLAT images, counts must be below the :math:`\approx` 30000 ADUs. In the process of building the daily PTC (see below), the FLAT saturation is intended, and achieved at around the 123000 ADUs (with exposure time of about 25 seconds). The daily FLATS reach values of :math:`\approx` 68000 ADUs. If you see large deviations from these values, please report it. Check the calibration sets and their configurations per filter and grating.
```

Comment:
The RAW count rates of the BIAS and DARK frames are not actually at 10 ADUs. Those are the count rates that show up in the images displayed in rubinTV after BIAS subtraction and DARK correction have been applied to the BIAS and DARK frames. I think it's best practice to still have the team check these things in rubinTV, but you should adjust this to say:

"The calibration images displayed in rubinTV are post-ISR images and should have BIAS and DARK corrections applied. This means that BIAS and DARK images should display with maximum count rates of about 10 ADUs."

Reply:
Thank you for the explanation.